src/features/claims/claim-details/components/ClaimInformation.tsx

```diff
@@ -105,6 +110,9 @@ export const ClaimInformation: React.FC<{
   memberId: string
   focus: boolean
 }> = ({ claimId, memberId, focus }) => {
+  const [creatingCoInsured, setCreatingCoInsured] = useState(false)
```

Comment:
An idea maybe? Then we could just use one variable after
```suggestion
  const [hasCoInsured, setHasCoInsured] = useState(!!coInsured)
```